doc/api/magic-plot.md

```diff
@@ -152,10 +152,11 @@ generate histograms without explicitly removing NULL entries.
 `%sqlplot` returns a `matplotlib.Axes` object.
 
 ```{code-cell} ipython3
-ax = %sqlplot histogram --table penguins.csv --column body_mass_g 
+ax = %sqlplot histogram --table penguins.csv --column body_mass_g
```

Comment:
Please revert these changes if not part of the issue

Reply:
revert it now